src/include/parser/expression/column_value_expression.h

```diff
@@ -113,7 +151,9 @@ class ColumnValueExpression : public AbstractExpression {
   }
 
   bool operator==(const AbstractExpression &rhs) const override {
-    if (!AbstractExpression::operator==(rhs)) return false;
```

Comment:
Add this line back.

Reply:
As discussed during the team meeting today, this was removed to ensure that when comparing ColumnValueExpressions, unnecessary fields are not compared. This is done in order for binding CVEs (output from a table) to succeed.